docs/source/infrastructure.rst

```diff
@@ -119,18 +119,19 @@ Pushing to Production
+1. `File a bug <https://bugzilla.mozilla.org/enter_bug.cgi?assigned_to=dlactin%40mozilla.com&bug_file_loc=http%3A%2F%2F&bug_ignored=0&bug_severity=normal&bug_status=NEW&bug_type=task&cc=dlactin%40mozilla.com&cc=jbuckley%40mozilla.com&cc=jcristau%40mozilla.com&cc=gbrown%20mozilla.com&cf_fx_iteration=---&cf_fx_points=---&cf_status_firefox77=---&cf_status_firefox78=---&cf_status_firefox79=---&cf_status_firefox80=---&cf_status_firefox_esr68=---&cf_status_firefox_esr78=---&cf_tracking_firefox77=---&cf_tracking_firefox78=---&cf_tracking_firefox79=---&cf_tracking_firefox80=---&cf_tracking_firefox_esr68=---&cf_tracking_firefox_esr78=---&cf_tracking_firefox_relnote=---&cf_tracking_firefox_sumo=---&comment=Balrog%20version%20X.Y%20is%20ready%20to%20be%20pushed%20to%20prod.%20Please%20deploy%20the%20new%20Docker%20images%20%28vX.Y%29%20for%20admin%2C%20public%2C%20and%20the%20agent.%0D%0A%0D%0AWe%27d%20like%20the%20production%20push%20for%20this%20to%20happen%20today.&component=Operations%3A%20Balrog&contenttypemethod=list&contenttypeselection=text%2Fplain&defined_groups=1&filed_via=standard_form&flag_type-37=X&flag_type-607=X&flag_type-708=X&flag_type-721=X&flag_type-737=X&flag_type-748=X&flag_type-787=X&flag_type-800=X&flag_type-803=X&flag_type-846=X&flag_type-864=X&flag_type-936=X&flag_type-941=X&flag_type-945=X&form_name=enter_bug&maketemplate=Remember%20values%20as%20bookmarkable%20template&op_sys=Unspecified&priority=--&product=Cloud%20Services&rep_platform=Unspecified&short_desc=please%20deploy%20balrog%20X.Y%20to%20prod&target_milestone=---&version=unspecified>`_ to have the new version pushed to production
 
   * Make sure you substitute the version number and choose the correct options from the bug template.
 
-2. Once the push has happened, verify that the code was pushed to production by checking the __version__ endpoints on `the Admin <https://aus4-admin.mozilla.org/__version__>`_ and `Public <https://aus5.mozilla.org/__version__>`_ apps.
-3. Manually delete and recreate the "production-ui" tag & release on Github to push the new UI to production:
+2. Before the deploy, make sure no release activity is ongoing, and notify `#releaseduty:mozilla.org` and `#sheriffs:mozilla.org` on matrix so they know to quickly escalate any fallout.
```

Comment:
This is still a bit vague.

1. We have already submitted the bug and will not actually be doing the deploy. Is this really, "when cloudops says they are about to do the deploy, make sure they wait while you ..."?
2. To make sure no release activity is ongoing, is that mostly asking on #releaseduty and #sheriffs?

Reply:
1. I guess.  Will tweak the language a bit more.
2. Yes.

Thanks.